nym-node/src/config/mod.rs

```diff
@@ -944,6 +945,7 @@ pub struct Wireguard {
 
     /// Tunnel port announced to external clients wishing to connect to the wireguard interface.
     /// Useful in the instances where the node is behind a proxy.
+    #[serde(alias = "announced_port")]
```

Comment:
This seems like it should be part of a separate PR, if it's a fix for something.